packages/flutter_tools/schema/material_fonts.json

```diff
@@ -0,0 +1,97 @@
+[
```

Comment:
Just a nit pick, but I probably would have used yaml here for easy of copy/pasting between this file and flutter.yaml.